internal/models/cluster/nodepool/nodepool_tkgs_volume.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package nodepool
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1CommonClusterTKGServiceVsphereVolume TKGServiceVsphereVolume defines a Persistent Volume Claim attached for the workload cluster.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.common.cluster.TKGServiceVsphereVolume
+type VmwareTanzuManageV1alpha1CommonClusterTKGServiceVsphereVolume struct {
+
+	// Volume capacity is in gib.
+	Capacity float32 `json:"capacity,omitempty"`
```

Comment:
I don't see the schema for these optional entries we can enter for the Nodepools, please add the same. We should provide this for both the TKGs and TKGm.

Note: Adding node pool is supported for TKGm workload clusters with version greater then 1.4.1

Reply:
Please check the same for AWS workload cluster

Reply:
As discussed in our call, the `volumes` spec has been added to the node pools. Also, TKGm vSphere and AWS doesn't have these fields in the spec.

Reply:
TKGm vSphere above 1.4.1 doesn't have these fields?

Reply:
Okay, we don not have such fields in the model itself 👍